### PR TITLE
8331993: Add counting leading/trailing zero tests for Integer

### PR DESCRIPTION
**Pull request by Hamlin-Li**
Hi,
Can you help to review the patch adding some test?
Currently, in hotspot/jtreg/compiler/vectorization/TestNumberOfContinuousZeros.java, there is only tests for Long, not for Integer.
Thanks.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8331993](https://bugs.openjdk.org/browse/JDK-8331993): Add counting leading/trailing zero tests for Integer (**Enhancement** - P4)

### Reviewers
 * [Christian Hagedorn](https://openjdk.org/census#chagedorn) (@chhagedorn - **Reviewer**) ⚠️ Review applies to [c0aaa35d](https://git.openjdk.org/jdk/pull/19154/files/c0aaa35de789e855dc6d28f95e615a6b4cef2eb5)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/19154/head:pull/19154` \
`$ git checkout pull/19154`

Update a local copy of the PR: \
`$ git checkout pull/19154` \
`$ git pull https://git.openjdk.org/jdk.git pull/19154/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 19154`

View PR using the GUI difftool: \
`$ git pr show -t 19154`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/19154.diff">https://git.openjdk.org/jdk/pull/19154.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/19154#issuecomment-2102465161)

### CODE REVIEW

File: test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java [hunk 1]
```diff
@@ -1164,11 +1164,21 @@ public class IRNode {
...
+    static {
+        vectorNode(COUNTTRAILINGZEROS_VI, "CountTrailingZerosV", TYPE_INT);
+    }
+
     public static final String COUNTLEADINGZEROS_VL = VECTOR_PREFIX + "COUNTLEADINGZEROS_VL" + POSTFIX;
     static {
         vectorNode(COUNTLEADINGZEROS_VL, "CountLeadingZerosV", TYPE_LONG);
     }
 
+    public static final String COUNTLEADINGZEROS_VI = VECTOR_PREFIX + "COUNTLEADINGZEROS_VI" + POSTFIX;
```
Comment (chhagedorn) [inline test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java:1177]: Would have been better to add `_` like that: `COUNT_LEADING_ZEROS_VI`

But the existing `IRNode` strings for the long versions already miss that. If you want to also fix this here, feel free to do so.

**Review by chhagedorn — APPROVED** [review]
Otherwise, looks good!

File: test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java (hunk 1 above)
Comment (Hamlin-Li) [reply test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java]: Yes, it's more readable. Fixed.
Thanks for your reviewing.

**Pull request closed without merging**